app/helpers/helper-functions.js

```diff
@@ -12,4 +14,35 @@ function setLabelData (data, labelData) {
   })
 }
 
-module.exports = { isChecked, setLabelData }
+function getPostCodeHtml (postcodeData, error) {
+  const postcode = postcodeData && postcodeData !== null ? postcodeData : ''
```

Comment:
const postcode = postcodeData || ''

Reply:
it saves null as value and does not come as undefined. I used it when a null is assigned to postcodeData to replace it with a ' ' .

Reply:
exactly what I meant. 
const postcode = postcodeData || '' 
means ==>
If null is supplied to postcodeData, then postcode becomes '' instead of postcodeData